decidim-dev/lib/decidim/dev/common_rake.rb

```diff
@@ -5,7 +5,7 @@
 require "decidim/dev"
 
 RSpec::Core::RakeTask.new(:spec) do |t|
-  t.rspec_opts = "--format progress --format RspecJunitFormatter -o ~/rspec/rspec.xml" if ENV["CI"]
+  t.rspec_opts = "--format doc --format RspecJunitFormatter -o ~/rspec/rspec.xml" if ENV["CI"]
```

Comment:
Move this back to the old value?

Reply:
It was useful when decidim-generators were lacking feedback on failure and we were not able to know which test failed.
I can re-set `progress` if you want, I see that now circleci is generating very long logs. We can always re-set doc format if we need.
Ok, I'm applying the change, no need to respond.